content/influxdb/v2.0/tools/grafana.md

```diff
@@ -73,30 +73,32 @@ configure your InfluxDB connection:
 2. [Configure your InfluxDB connection](#configure-your-influxdb-connection)
 
 ### Set up InfluxDB compatibility
-To query InfluxDB OSS 2.0 with InfluxQL, the following must exist:
-
-- **Valid authentication credentials** (either of the following)
-  - **v2 authentication token**  
-    InfluxDB OSS 2.0 [authentication token](/influxdb/v2.0/security/tokens/).
-  - **v1 compatible authentication credentials**  
-    InfluxDB OSS 2.0 provides a [1.x compatible authentication API](#view-and-create-influxdb-v1-authorizations)
-    that let's you authenticate with a username and password like InfluxDB 1.x
-    _(separate from the credentials used to log into the InfluxDB user interface)_.
-- **Database and retention policy (DBRP) mappings**  
-  When using InfluxQL to query InfluxDB 2.0, the query must specify a database and a retention policy.
-  DBRP mappings associate database and retention policy combinations with InfluxDB 2.0 [buckets](/influxdb/v2.0/reference/glossary/#bucket).
-
-When upgrading from InfluxDB 1.x to 2.0 using the [official upgrade process](/influxdb/v2.0/upgrade/v1-to-v2/),
-InfluxDB creates v1 compatible authorizations for all _non-admin_ 1.x users and all necessary DBRP mappings.
-Grafana dashboards should work against a upgraded database without change.
-
-If you manually migrated from InfluxDB 1.x to InfluxDB 2.0 or are starting with
-a fresh 2.0 installation, you **must [manually create any necessary DBRP mappings](#view-and-create-influxdb-dbrp-mappings)**.
-**We recommend using token authentication** unless your Grafana configuration already
-is already configured to authenticate with username and password.
+To query InfluxDB OSS 2.0 with InfluxQL, you must successfully authenticate with
+InfluxDB and database/retention policy (DBRP) mappings must exist.
+
+- **Are you starting with a fresh InfluxDB 2.0 instance?**
+  - **Authentication:** use [InfluxDB v2 token authentication](/influxdb/v2.0/security/tokens/).
+  - **DBRP mappings:** [manually create DBRP mappings](#view-and-create-influxdb-dbrp-mappings).
+
+- **Did you upgrade from InfluxDB 1.x to 2.0 using the [official upgrade process](/influxdb/v2.0/upgrade/v1-to-v2/)?**
+  - **Authentication:** use the _non-admin_ [v1 compatible authentication credentials](#view-and-create-influxdb-v1-authorizations)
+    created for in the upgrade process.
+  - **DBRP mappings:** InfluxDB automatically created DBRP mappings in the upgrade process.
+
+- **Did you manually migrate from InfluxDB 1.x to 2.0?**
+  - **Authentication:** If your InfluxDB 1.x instance required authentication,
+    [create v1 compatible authentication credentials](#view-and-create-influxdb-v1-authorizations)
+    to match your previous 1.x username and password.
+    Otherwise, use [InfluxDB v2 token authentication](/influxdb/v2.0/security/tokens/).
+  - **DBRP mappings:** [manually create DBRP mappings](#view-and-create-influxdb-dbrp-mappings).
```

Comment:
```suggestion
## Configure Grafana to use InfluxQL

To query InfluxDB OSS 2.0 with InfluxQL, find your use case below, and then complete the instructions to configure Grafana:

- [Installed a new InfluxDB 2.0 instance](#new-install-of-influxdb-2-0)
- [Configure Grafana for pgraded from InfluxDB 1.x to 2.0 (following the official upgrade)](#upgrade-from-influxdb-1-x-to-2-0)
- [Manually migrated from InfluxDB 1.x to 2.0](#manually-migrated-from-influxdb-1-x-to-2-0)

### Installed a new InfluxDB 2.0 instance

To configure Grafana to use InfluxQL with a new install of InfluxDB 2.0, do the following:

1. [Authenticate with InfluxDB 2.0 tokens](/influxdb/v2.0/security/tokens/).
2. [Manually create DBRP mappings](#view-and-create-influxdb-dbrp-mappings).
3. [Configure your InfluxDB connection](#configure-your-influxdb-connection).

### Upgrade from InfluxDB 1.x to 2.0

To configure Grafana to use InfluxQL when you've upgraded from InfluxDB 1.x to InfluxDB 2.0 (following the official upgrade), authenticate using the _non-admin_ [v1 compatible authentication credentials](#view-and-create-influxdb-v1-authorizations). InfluxDB automatically created DBRP mappings in the upgrade process.

### Manually migrate from InfluxDB 1.x to 2.0

To configure Grafana to use InfluxQL when you've manually migrated from InfluxDB 1.x to InfluxDB 2.0, do the following:

1. If your InfluxDB 1.x instance required authentication,
    [create v1 compatible authentication credentials](#view-and-create-influxdb-v1-authorizations)
    to match your previous 1.x username and password.
    Otherwise, use [InfluxDB v2 token authentication](/influxdb/v2.0/security/tokens/).
2. [Manually create DBRP mappings](#view-and-create-influxdb-dbrp-mappings).

```

Reply:
Scott, what do you think about this structure (we may need to tweak wording for steps a bit)? It highlights each use case and has step-by-step for each.